src/ophys_etl/decrosstalk/decrosstalk.py

```diff
@@ -112,6 +112,9 @@ def unmix_all_ROIs(raw_roi_traces: dc_types.ROISetDict,
 
     Returns
     -------
+    A boolean indicating whether or not we were able to successfully
+    unmix the ROIs in this input ROISetDict
+
```

Comment:
It would be nice to have a
```
Notes
-----
```
section describing briefly the situation where the boolean is `True` or `False`